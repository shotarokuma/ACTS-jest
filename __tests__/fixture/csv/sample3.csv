Column1	Column2	Column3	Column4	Column5	Column6	Column7	Column8	Column9	Column10
true	false								true