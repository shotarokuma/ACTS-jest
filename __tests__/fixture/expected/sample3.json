[
{"Column1":"true","Column2":"false","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"true"}
]
